src/main/java/com/example/onlinebookstore/model/Order.java

```diff
@@ -0,0 +1,56 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(exclude = {"user", "orderItems"})
+@EqualsAndHashCode(of = {"id", "status", "total", "orderDate", "shippingAddress"})
```

Comment:
What is 10 more fields will be added to order?
I suggest to use annotation to exclude some specific fields (https://www.baeldung.com/java-lombok-equalsandhashcode), I would be better than write what fields to include, because it is ofter more fields to include than exclude

Reply:
tnx for advice.
done